mep3_driver/package.xml

```diff
@@ -20,6 +20,7 @@
   <depend>sensor_msgs</depend>
   <depend>python3-can</depend>
   <depend>python3-gpiozero</depend>
+  <depend>rpi_lcd</depend>
```

Comment:
Ovaj pakem msm da ne postoji. Dostupni Python paketi su ovdje:
https://github.com/ros/rosdistro/blob/master/rosdep/python.yaml

To cemo manualno instalirati